app/javascript/admin/controllers/landing_page_modal_controller.js

```diff
@@ -0,0 +1,22 @@
+import ModalController from './modal_controller';
+
+export default class LandingPageModalController extends ModalController {
+  static targets = ['overwrite', 'landingPageCheckbox'];
+
+  openModal() {
```

Comment:
At the moment we trigger this code on checkbox click, regardless of whether there is an existing lock screen or not, and it causes an error in the JS when there isn't a current lock screen (because the modal isn't added in the html in this case). 

There's no real impact on the user at the moment (it just quietly errors in the console), but Cypress will fail because of the unhandled exception, and we'll want to tidy it up. I suspect all we need to do is exit this method straight away if there is no current lock screen.